src/index.js

```diff
@@ -6,7 +6,7 @@ import { ApolloProvider } from "@apollo/react-hooks";
 import "./index.css";
 import App from "./App";
 import * as serviceWorker from "./serviceWorker";
-import { loofahApiUrl } from "./config/config";
+import { loofahApiUrl } from "./Config/config";
```

Comment:
Could ya rebase and change back to lowercase? If you're still having issues let me know

Reply:
done 👍